.github/workflows/_featurenet-create-from-branch.yml

```diff
@@ -0,0 +1,155 @@
+---
+name: FE - Create from branch
+
+on:
+  workflow_call:
+    inputs:
+      featurenet-name:
+        description: 'Name of featurenet'
+        type: string
+        required: true
+      expiration:
+        description: 'Time after which featurenet will be removed'
+        required: false
+        type: string
+        default: '48h'
+      replicas:
+        description: 'Number of replicas to start, from 0 to 50'
+        required: false
+        default: '5'
+        type: string
+      internal:
+        description: 'Internal network, accessible from VPN only'
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  validate-inputs:
+    name: Validate inputs
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Validate inputs
+        shell: bash
+        # yamllint disable rule:line-length
+        run: |
+          if [[
+          ! "${{ inputs.featurenet-name }}" =~ ^[a-z0-9][a-z0-9\-]{4,48}$
+          ]]
+          then
+            echo "!!! Invalid featurenet name"
+            exit 1
+          fi
+        # yamllint enable rule:line-length
+
+  get-final-names:
```

Comment:
I guess we do not need that job now, as we do not prefix `updatenet` anywhere. So plain `${{ inputs.featurenet-name }}` would suffice; this comment applies to other files as well